dev.zed.Zed.yaml

```diff
@@ -0,0 +1,105 @@
+# dev.zed.Zed.yaml
+app-id: dev.zed.Zed
+runtime: org.freedesktop.Platform
```

Comment:
You may need to change this to `org.freedesktop.Sdk` instead, so you have access to Git, and some other tools.

Reply:
I thought by defining `org.freedesktop.Sdk` as an `SDK` option was enough, but I'll gladly change the runtime

Reply:
No, the SDK used at build time only, runtime is used at run time.

Reply:
Thanks -- I'll keep it `org.freedesktop.Platform` then. The build runs with both

Reply:
Dependencies added!